tests/integ/sagemaker/serve/test_schema_builder.py

```diff
@@ -202,6 +202,33 @@ def test_model_builder_happy_path_with_task_provided_remote_schema_mode(
                 ), f"{caught_ex} was thrown when running transformers sagemaker endpoint test"
 
 
+@pytest.mark.skipif(
+    PYTHON_VERSION_IS_NOT_310,
+    reason="Testing Schema Builder Simplification feature - Remote Schema",
+)
+@pytest.mark.parametrize(
+    "model_id, task_provided, instance_type_provided",
+    [("openai/whisper-tiny.en", "automatic-speech-recognition", "ml.m5.4xlarge")],
```

Comment:
Will this instance be available in all regions that this test will be executed?

Reply:
I think so I got it from the allowlisted instances list